resources/chefdk/msi/localization-en-us.wxl.erb

```diff
@@ -20,4 +20,9 @@
   <String Id="FeaturePSModuleName"><%= friendly_name %> PowerShell wrappers</String>
   <String Id="FeatureChefDkEnvHacks">ChefDK Environment Customizations</String>
   <String Id="FeatureChefDkEnvHacksDesc">Allows ChefDK to try to find a suitable HOME drive when running Ruby programs.</String>
+  
+  <String Id="ChefDkStartMenuShortcutDefName"><%= friendly_name %></String>
+  <String Id="ChefDkStartMenuShortcutDefDescription"><%= friendly_name %> Command Prompt</String>
```

Comment:
In the [WIX docs](http://wixtoolset.org/documentation/manual/v3/xsd/wix/shortcut.html), it's the shortcut description. I can't figure out how it actually affects the shortcut, but it's in the shortcut tutorials.